src/Umbraco.Web.UI.Client/src/packages/documents/documents/item/repository/types.ts

```diff
@@ -17,10 +17,21 @@ export interface UmbDocumentItemModel {
 	parent: { unique: UmbEntityUnique } | null; // TODO: Use UmbReferenceByUnique when it support unique as null
 	unique: string;
 	variants: Array<UmbDocumentItemVariantModel>;
+	flags?: Array<UmbEntityFlag>;
+}
+
+export interface UmbDocumentItemWithFlagsModel extends UmbDocumentItemModel {
```

Comment:
Is there any case where a Document Item won't have flags, since we need this specific interface instead of adding it directly to the item model?